packages/webpack-config-terra/package.json

```diff
@@ -65,8 +65,7 @@
     "@babel/core": "^7.11.1",
     "@cerner/eslint-config-terra": "^5.6.0",
     "@cerner/jest-config-terra": "^1.7.0",
-    "eslint": "^7.32.0",
-    "jest": "26.6.3"
```

Comment:
Doesn't webpack config terra have jest tests?

Reply:
Yes. The reason this was removed is because it's hardcoded to Jest 26. Our options here are either to remove the `jest` devDependency or upgrade it to v27 - I removed it since the dependency is automatically installed, but I'm fine with bumping the version if we prefer.

Reply:
Yeah, the dependency will get installed automatically anyway. We can remove it so that there's less maintenance.